### PR TITLE
ADD ADDVOW TOKEN

### PR DESCRIPTION
**Pull request by garudawork**

### CODE REVIEW

**Comment by StrathCole** [comment]
It seems you did not prior merge the main *correctly* as your change would remove one pair and replace it.

**Comment by garudawork** [comment]
> It seems you did not prior merge the main _correctly_ as your change would remove one pair and replace it.

done remove one pair

File: columbus-5/swap/terraport/pairs.json [hunk 1]
```diff
@@ -369,9 +369,9 @@
         "type": "native"
     },
     "asset_two": {
-        "denom": "ibc/E1E3674A0E4E1EF9C69646F9AF8D9497173821826074622D831BAB73CCB99A2D",
```
Comment (StrathCole) [inline columbus-5/swap/terraport/pairs.json:372]: You're removing an existing pair here instead of adding yours. That's not correct.

Comment (garudawork) [reply columbus-5/swap/terraport/pairs.json:372]: already added back

**Pull request merged**